src/dotnet/commands/dotnet-msbuild/MSBuildForwardingApp.cs

```diff
@@ -77,6 +79,11 @@ internal static CommandOption AddVerbosityOption(CommandLineApplication app)
             return app.Option("-v|--verbosity", LocalizableStrings.VerbosityOptionDescription, CommandOptionType.SingleValue);
         }
 
+        private static string Escape(string arg) =>
```

Comment:
Why escaping this here instead of at the dotnet restore command? Are you doing this in anticipation of other cases like this?